assets/js/markmap.js

```diff
@@ -1,19 +1,15 @@
-{{ with .Site.Params.markmap }}
-{{ if .enable }}
-
-
-(function($) {
-    var needMarkmap = false;
+{{ if lt hugo.Version "0.93.0" -}}
     $('.language-markmap').parent().replaceWith(function() {
-        needMarkmap = true;
-        return $('<div class="markmap">').text($(this).text());
+        return $('<pre class="markmap">').text($(this).text());
     });
-
-    const { markmap } = window;
-    if(needMarkmap) {
-        markmap.autoLoader.renderAll();
-    }
-
-})(jQuery);
-{{ end }}
-{{ end }}
+{{ end -}}
```

Comment:
Have you tested this code for use with Hugo <0.93.0? Why the switch from `div` to `pre`?

As you know we don't officially support older versions of Hugo, and certainly not <0.93. I'd suggest removing this code altogether.

Reply:
Remove the code targeting at hugo versions <0.93.0 alltogether.